docs/src/components/action/ArrowButton.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import IconButton, { IconButtonProps } from '@material-ui/core/IconButton';
+import KeyboardArrowLeftRounded from '@material-ui/icons/KeyboardArrowLeftRounded';
+import KeyboardArrowRightRounded from '@material-ui/icons/KeyboardArrowRightRounded';
+
+export default function ArrowButton({
+  direction,
+  ...props
+}: { direction: 'left' | 'right' } & IconButtonProps) {
+  const label = {
+    left: 'Previous',
+    right: 'Next',
+  };
+  return (
+    <IconButton
+      size="small"
+      aria-label={label[direction]}
+      {...props}
+      sx={{
+        color: '#fff',
```

Comment:
Is this light/dark mode safe? Should we use some theme token instead?

Reply:
Good point, currently the design has the usage only on the dark background but I can add my guess on the light background as well. will update it.